src/cli/src/cluster/install/local.rs

```diff
@@ -14,24 +14,37 @@ use super::get_binary;
 use super::CommandUtil;
 
 pub async fn install_local(opt: InstallCommand) -> Result<(), CliError> {
+    use std::path::Path;
+    use std::fs::create_dir_all;
+
+    let log_dir;
+    if std::env::consts::OS == "macos" {
```

Comment:
should use Rust's cfg feature to target specific OS:  https://doc.rust-lang.org/stable/rust-by-example/attribute/cfg.html.  You can example for macOS: https://github.com/infinyon/future-aio/blob/master/src/zero_copy.rs